analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<p>I would recommend having sections like this:</p><ol><li>Read the census data and do any necessary pre-processing, including selecting for only the columns we care about. Print a few of the rows so that users know what kind of data it contains.</li><li>Read the ACS demographic data and do any pre-processing. Print a few of the rows so that users know what kind of data it contains.</li><li>Join the data. Print a few of the rows so that users know what kind of data it contains.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks Nich for a comprehensive feedback! Incorporated all the suggestions. </p>